data_structure.txt

```diff
@@ -0,0 +1,15 @@
+The order:
+[
+    {
+        Name : "String"
+        price : number
+        description : "String"
+        quantity : number
+        size : {'string' : $} , ... : {"medium": 5} -> object{}
+        topping options : { Boba: 0.75,...} [{Boba: 0.75},{Almond: 1.00}] -> array[object{}]
+        milk/base options
+    },
+    {
+      ...
+    } ,...
+]

```

Comment:
Please remove, note personally or document somewhere like Jira instead of pushing jibberish info directly into the repo.
Create the proposed structure or an actual TypeScript props in `src/components/BobaContainer/bobalist.jsx` which is more meaningful